src/app/bug/bug.component.spec.ts

```diff
@@ -47,17 +50,21 @@ describe('BugComponent', () => {
     component = spectator.component;
     bugService = spectator.inject(BugService);
     bugService.state = new BehaviorSubject<any>([]);
+    routerMock = spectator.inject(Router);
 
   });
 
-  it('should be createt', () => {
+  it('should be created', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8891152)
```suggestion
  it("should be created", () => {
```